src/Microsoft.PowerShell.MarkdownRender/LinkInlineRenderer.cs

```diff
@@ -19,11 +19,11 @@ protected override void Write(VT100Renderer renderer, LinkInline obj)
             // Format link as image or link.
             if (obj.IsImage)
             {
-                renderer.Write(renderer.EscapeSequences.FormatImage(obj.FirstChild.ToString()));
+                renderer.Write(renderer.EscapeSequences.FormatImage(obj.FirstChild?.ToString()));
             }
             else
             {
-                renderer.Write(renderer.EscapeSequences.FormatLink(obj.FirstChild.ToString(), obj.Url));
+                renderer.Write(renderer.EscapeSequences.FormatLink(obj.FirstChild?.ToString(), obj.Url));
```

Comment:
We check `obj.FirstChild != null` in both code path in the "if-else" - seems we should move the check to new "if":
```c#
if (obj.FirstChild != null)
{
    if (obj.IsImage)
    {
        renderer.Write(renderer.EscapeSequences.FormatImage(obj.FirstChild.ToString()));
    }
    else
    {
        renderer.Write(renderer.EscapeSequences.FormatLink(obj.FirstChild.ToString(), obj.Url));
    }
}

Reply:
The question marks here would have different semantics of course, but I imagine `FormatLink` will thrown an NRE if its first argument is `null`.

So I would do similar to @iSazonov (just a style change):
```csharp
if (obj.FirstChild == null)
{
    return;
}

if (obj.IsImage)
{
    renderer.Write(renderer.EscapeSequences.FormatImage(obj.FirstChild.ToString()));
}
else
{
    renderer.Write(renderer.EscapeSequences.FormatLink(obj.FirstChild.ToString(), obj.Url));
}
```

Reply:
We support having empty alt-text for images and empty link text. So updated accordingly.